exporter/sumologicexporter/prometheus_formatter.go

```diff
@@ -55,11 +55,11 @@ func newPrometheusFormatter() (prometheusFormatter, error) {
 }
 
 // PrometheusLabels returns all attributes as sanitized prometheus labels string
-func (f *prometheusFormatter) tags2String(attr pdata.AttributeMap, labels pdata.StringMap) prometheusTags {
+func (f *prometheusFormatter) tags2String(attr pdata.AttributeMap, labels pdata.AttributeMap) prometheusTags {
 	mergedAttributes := pdata.NewAttributeMap()
 	attr.CopyTo(mergedAttributes)
-	labels.Range(func(k string, v string) bool {
-		mergedAttributes.UpsertString(k, v)
+	labels.Range(func(k string, v pdata.AttributeValue) bool {
+		mergedAttributes.UpsertString(k, v.StringVal())
```

Comment:
TODO link to https://github.com/open-telemetry/opentelemetry-collector/issues/3815